### ISSUE TITLE
AWS PostgreSQL DB Instance

### ISSUE THREAD

**Issue by francisrod01** [imperialwicket/imperialwicket.com#3]
Today exists the AWS PostgreSQL DB Instance. It's more easy to use.

Creating a PostgreSQL DB Instance and Connecting to a Database on a PostgreSQL DB Instance

http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html

**Comment by imperialwicket** [comment]
Yes, you can definitely go the RDS route now, and there are easier ways to install on your own EC2 instances as well (the postgresql install posts are from 2011). 

Perhaps it's time to start more clearly archiving some legacy content...
